tailwind.config.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
         automobile: "repeat(auto-fit, minmax(300px, 1fr))"
       },
       screens: {
+        sm: "320px",
+        md: "640px",
+        xmd: "840px",
```

Comment:
Is there a reason why you added custom screen sizes?

Reply:
yea for better responsiveness

Reply:
> yea for better responsiveness

Could you please elaborate a bit more on why this is better for responsiveness please?